data/decks/tinker.json

```diff
@@ -0,0 +1,172 @@
+{
+	"id": "tinker",
+	"season": 4,
+	"display_name": "Tinker Knight",
+	"cardback": "cardback_tinker.jpg",
+	"exceed_cost": -1,
+	"wide_animation": "tinker_tank",
+	"starting_life": 15,
+	"on_death": {
+		"condition": "not_exceeded",
+		"effect_type": "exceed_now",
+		"and": {
+			"effect_type": "set_used_character_bonus"
+		}
+	},
+	"on_exceed": {
+		"effect_type": "set_life_per_gauge",
+		"amount": 4,
+		"maximum": 20
+	},
+	"ability_effects": [
+	],
+	"exceed_ability_effects": [
+		{
+			"NOTE": "If it happened on our turn",
+			"timing": "end_of_turn",
+			"condition": "used_character_bonus",
+			"effect_type": "reset_character_positions",
```

Comment:
i think these things need strings in card_definitions.gd if they can be ordered by the player via choices in cleanup

Reply:
I was betting on tinker not having any other cleanup effects, but probably best to be safe